src/index.js

```diff
@@ -60,13 +65,18 @@ import {
   StyleLayoutPanel,
   StyleLegendPanel,
   StyleNotesPanel,
+  StyleShapesPanel,
+  StyleImagesPanel,
   StyleTracesPanel,
 } from './default_panels';
```

Comment:
same, can we add these to the README

Reply:
Hmmm well none of the other default panels are in the readme, so I would vote no...

Reply:
I mean the positionningRef, the Accordions, the connect functions : )